packages/worker/src/workflow/threaded-vm.ts

```diff
@@ -180,6 +180,7 @@ export class VMWorkflowThreadProxy implements Workflow {
     workerThreadClient: WorkerThreadClient,
     options: WorkflowCreateOptions
   ): Promise<VMWorkflowThreadProxy> {
+    delete (options.info.unsafe as any).now;
```

Comment:
@bergundy how's this?

Reply:
👍 just please add a comment explaining the cast